framework/utils/utils.py

```diff
@@ -222,23 +222,44 @@ def convertMultipleToBytes(sizeString):
     except:
       raise IOError(UreturnPrintTag('UTILITIES')+': ' +UreturnPrintPostTag('ERROR') + '->  can not understand how to convert expression '+str(sizeString)+' to number of bytes. Accepted Mb,Gb,Kb (no case sentive)!')
 
-def stringsThatMeanTrue():
+# I don't think there's a reason to make this an enum, but it could be done.
+trueThingsFull = ('True', 'Yes', '1')
+trueThings = tuple(x[0].lower() for x in trueThingsFull)
+trueThingLegacy = ['yes','y','true','t','si','vero','dajie','oui','ja','yao','verum', 'evet', 'dogru', '1', 'on']
+
+def stringIsTrue(s):
   """
-    Return list of strings with the meaning of true in RAVEN (eng,ita,roman,french,german,chinese,latin, turkish, bool)
-    @ In, None
-    @ Out, listOfStrings, list, list of strings that mean True in RAVEN
+    Determines if provided entity corresponds to a truth statement
+    @ In, s, string or castable, entity to check
+    @ Out, stringIsTrue, bool, True if string is recognized by RAVEN as evaluating to True
   """
-  listOfStrings = list(['yes','y','true','t','si','vero','dajie','oui','ja','yao','verum', 'evet', 'dogru', '1', 'on'])
-  return listOfStrings
+  # as far as I know, nothing in python cannot be cast as a string.
+  s = str(s).strip()
+  return s.lower().startswith(trueThings)
```

Comment:
is lower explicitly entered in the functions. If yes, discard my previous comments about it

Reply:
As you can see here, we handle the `lower` within the method.

Reply:
So we are using somebody else to merge it? Or should I do that?

Reply:
If you want to perform the PR review, as a trained RAVEN developer you are authorized to do that. Make sure you go through the checklist in the PR header.